charts/wsj2023-access/values/production.yaml

```
@@ -1,3 +1,4 @@
 ---
 domains:
   - wsj2023-access.production.webservices.scouterna.net
+oauth-plugin: office-oauth-wsj2023-access
```

I didn't create the app registration for production yet, so either do it like this and it will fail until that's setup, or share config with staging for the time being:

```suggestion
oauth-plugin: office-oauth-wsj2023-access-production
```